mlrun/config.py

```diff
@@ -961,6 +961,10 @@ def resolve_chief_api_url(self) -> str:
         self.httpdb.clusterization.chief.url = chief_api_url
         return self.httpdb.clusterization.chief.url
 
+    @staticmethod
+    def internal_labels():
+        return ["kind", "host"]
+
```

Comment:
the work here is to add all internal labels to a py file of with constants.
make sure all the label values use the file / field above.

there are many more labels (names) such as "job-type", "workflow", "type", "project" , "mlrun/username", "mlrun/username_domain" and pretty much every label mlrun adds that is not added by *users*. 
start by searching their name and this pr is expected to be a small refactor